docs/src/testbeds.md

```diff
@@ -53,19 +52,19 @@ github:danth/stylix
     └───x86_64-linux
         ├───docs: package 'stylix-book'
         ├───palette-generator: package 'palette-generator'
-        ├───testbed-gnome-dark: package 'testbed-gnome-dark'
-        ├───testbed-gnome-light: package 'testbed-gnome-light'
-        ├───testbed-kde-dark: package 'testbed-kde-dark'
-        └───testbed-kde-light: package 'testbed-kde-light'
+        ├───testbed-gnome-default-dark: package 'testbed-gnome-default-dark'
+        ├───testbed-gnome-default-light: package 'testbed-gnome-default-light'
+        ├───testbed-kde-default-dark: package 'testbed-kde-default-dark'
+        └───testbed-kde-default-light: package 'testbed-kde-default-light'
 ```
 
 (This has been edited down to only the relevant parts.)
 
 To start a testbed, each of which is named in the format
-`testbed-«target»-«polarity»`, run the following command:
+`testbed-«target»-«name»-«polarity»`, run the following command:
 
 ```console
-user@host:~$ nix run .#testbed-«target»-«polarity»
+user@host:~$ nix run .#testbed-«target»-«name»-«polarity»
```

Comment:
`«name»` is used here, but `«testbed»` is used above (currently line 20) - consider making this consistent and more specific by using `«testbed name»` everywhere.

Reply:
Should we keep `/modules/«module»/testbeds/«testbed».nix` from

https://github.com/danth/stylix/blob/e9c97fcdc9338f5603139300137f7d2f43c33d9e/docs/src/testbeds.md?plain=1#L20

and rename the testbed expression to `testbed-«module»-«testbed»-«polarity»`? That way we avoid the potentially wordy "testbed name" specifier by making "module" and "testbed" are explicitly distinct.

Reply:
That makes sense and also resolves the discussion at https://github.com/danth/stylix/pull/854#issuecomment-2655112038 by specifying that the testbed matches the module name and not target name.